lib/driver.js

```diff
@@ -170,6 +170,9 @@ const NO_PROXY_WEB_LIST = /** @type {import('@appium/types').RouteMatcher[]} */
 
 const MEMOIZED_FUNCTIONS = ['getStatusBarHeight', 'getDevicePixelRatio', 'getScreenInfo'];
 
+// Capabilities that do not have xcodebuild process
+const CAP_NAMES_NO_XCODEBUILD_REQUIRED = ['webDriverAgentUrl', 'usePreinstalledWDA'];
```

Comment:
Please also update isXcodebuildNeeded to use this constant

Reply:
ba98cc8a972904e7fe8fc38fc8f60672b68c908e